backend/de.metas.printing/de.metas.printing.base/src/main/sql/postgresql/system/80-de.metas.printing/5620501_sys_gh12221_AddExampleFilterForPDFConcatenation.sql

```diff
@@ -0,0 +1,15 @@
+-- 2022-01-04T15:54:42.557208500Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541440,'O',TO_TIMESTAMP('2022-01-04 17:54:42','YYYY-MM-DD HH24:MI:SS'),100,'U','Y','de.metas.printing.UserQuery_Example',TO_TIMESTAMP('2022-01-04 17:54:42','YYYY-MM-DD HH24:MI:SS'),100,'ItemName=''Rechnung'' AND Processed=''N''')
+;
+
+-- 2022-01-04T15:54:48.295671300Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_SysConfig SET ConfigurationLevel='O', IsActive='N',Updated=TO_TIMESTAMP('2022-01-04 17:54:48','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541440
+;
+
+-- 2022-01-04T16:01:11.743439900Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_SysConfig SET ConfigurationLevel='O', Description='This sys config is used de.metas.printing.model.validator.AsyncBatchNotificationHelper#enqueuePrintQueues for filter which printing queues to enqueue.',Updated=TO_TIMESTAMP('2022-01-04 18:01:11','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541440
```

Comment:
the description also needs to contain the information that
* it's about a whereclause
* which table
* which part is the fixed prefix and which part can be different on multiple sysconfigs.
* the info about AsyncBatchNotificationHelper is not really helpful for anyone outside this company
  * also, if i know the prfix, i can fix it myself, via FTS in my IDE